mitmproxy/console/common.py

```diff
@@ -359,6 +359,11 @@ def raw_format_flow(f, focus, extended):
         uc = "title"
 
     url = f["req_url"]
+
+    # TODO: Add a configuration setting that disables this behaviour?
```

Comment:
:-1: on this, I can hardly imagine a use case for this. No strong opinion though.